src/main/scala/no/ndla/articleapi/repository/ArticleRepository.scala

```diff
@@ -195,34 +279,45 @@ trait ArticleRepository {
     }
 
     override def documentsWithIdBetween(min: Long, max: Long): List[Article] =
-      articlesWhere(sqls"ar.id between $min and $max").toList
+      articlesWhere(sqls"ar.article_id between $min and $max").toList
 
     private def articleWhere(whereClause: SQLSyntax)(
         implicit session: DBSession = ReadOnlyAutoSession): Option[Article] = {
       val ar = Article.syntax("ar")
-      sql"select ${ar.result.*} from ${Article.as(ar)} where ar.document is not NULL and $whereClause"
```

Comment:
Var det meininga å fjerne sjekk på ar.document?

Reply:
Yes. 
Håndterer null-sjekken i `fromResultSet` istedet. Dette må til for å få til at dersom siste revision er null så får vi en 404.